proto/internal/dispatch/v1/dispatch.proto

```diff
@@ -19,6 +19,7 @@ message DispatchCheckRequest {
   enum DebugSetting {
     NO_DEBUG = 0;
     ENABLE_DEBUGGING = 1;
```

Comment:
maybe we want to rename this to BASIC debugging? it won't be a breaking change in the API.